src/gui/main.cpp

```diff
@@ -73,6 +73,9 @@ int main(int argc, char **argv)
 #endif
     OCC::Application app(argc, argv);
 
+    QTextCodec *codec1 = QTextCodec::codecForName("UTF-8");
```

Comment:
Does that really makes sense outside of Windows? AFAIK everyone else is using UTF-8 by default now.

Reply:
@er-vin Might be not-needed outside of Windows, so conditional compilation could help. On Windows, though, adding this - helps to make logs written via `qDebug`, `qCInfo`, and others, readable in the **.log** files later.

Reply:
I'm concerned about this though... because we don't change the locale but then we enforce a different codec than the "right" codec for that locale... If it's "just" about the log file because we want to make sure it's always encoded in UTF-8 (because indeed right now it will be encoded in whatever codec required by the user locale) then I'd only enforce UTF-8 on the codec used in the QTextStream of Logger. This would have way less side effects.

Reply:
And actually now that I think of it, this might be the only change you need... if I'm not mistaken, all QByteArrays written to the log file at that point would be forcibly interpreted as UTF-8.

Reply:
> And actually now that I think of it, this might be the only change you need... if I'm not mistaken, all QByteArrays written to the log file at that point would be forcibly interpreted as UTF-8.

@er-vin Works in most scenarios but fails for the SQLite case, so changes to `bindValue` and `qCInfo(lcDb) << "Updating file record for path:" << QString::fromUtf8(record._path) << "inode:" << record._inode` are still needed

Reply:
@er-vin Unless, I could go with changing QByteArray to QString for paths everywhere.

Reply:
> > And actually now that I think of it, this might be the only change you need... if I'm not mistaken, all QByteArrays written to the log file at that point would be forcibly interpreted as UTF-8.
> 
> @er-vin Works in most scenarios but fails for the SQLite case, so changes to `bindValue` and `qCInfo(lcDb) << "Updating file record for path:" << QString::fromUtf8(record._path) << "inode:" << record._inode` are still needed

OK, then I was likely wrong.

Reply:
> @er-vin Unless, I could go with changing QByteArray to QString for paths everywhere.

Well, that'd be a much bigger and error prone endeavor. I'd keep that for much later, master doesn't need this right now. ;-)